nextjs-app/app/_components/slider-module.tsx

```diff
@@ -11,28 +15,133 @@ type SliderModuleProps = {
+                className="w-full h-full object-cover" 
+                image={items[prevIndex].image} 
+              />
+            </div>
+          )}
+        </div>
+        
+        {/* Main slide */}
+        <div className="w-3/5 h-full relative overflow-hidden">
+          <AnimatePresence initial={false} mode="wait">
+            <motion.div
+              key={currentIndex}
+              initial={{ x: direction * 300, opacity: 0 }}
+              animate={{ x: 0, opacity: 1 }}
+              exit={{ x: direction * -300, opacity: 0 }}
+              transition={{ duration: 0.5 }}
+              className="w-full h-full"
+            >
+              {items[currentIndex]?.image && (
+                <div className="relative h-full">
+                  <Image 
+                    className="w-full h-full object-cover" 
+                    image={items[currentIndex].image} 
+                  />
+                </div>
+              )}
+            </motion.div>
+          </AnimatePresence>
+        </div>
+        
+        {/* Next slide thumbnail */}
+        <div 
+          className="w-1/5 cursor-pointer pl-2 h-full" 
+          onClick={nextSlide}
+        >
+          {items[nextIndex]?.image && (
+            <div className="relative h-full">
+              <div className="absolute inset-0 bg-black/40"></div>
+              <Image 
+                className="w-full h-full object-cover" 
+                image={items[nextIndex].image} 
+              />
+            </div>
           )}
         </div>
-      ))}
+      </div>
+
+      {/* Text content below the slider */}
+      <div className="mt-6 w-3/5 mx-auto">
+        <AnimatePresence initial={false} mode="wait">
+          <motion.div
+            key={currentIndex}
+            initial={{ y: 20, opacity: 0 }}
+            animate={{ y: 0, opacity: 1 }}
+            exit={{ y: -20, opacity: 0 }}
+            transition={{ duration: 0.3 }}
+          >
+            {items[currentIndex]?.heading && (
+              <h2 className="header text-24 md:text-36 lg:text-48 mb-2">
+                {items[currentIndex].heading}
+              </h2>
+            )}
+            {items[currentIndex]?.body?.length && (
+              <div className="text-16">
+                <PortableText value={items[currentIndex].body as PortableTextBlock[]} />
+              </div>
+            )}
+          </motion.div>
+        </AnimatePresence>
+      </div>
+
+      {/* Dots indicator */}
+      <div className="flex justify-center mt-6 gap-2">
+        {items.map((_, i) => (
+          <button
+            key={i}
+            onClick={() => {
+              setDirection(i > currentIndex ? 1 : -1);
+              setCurrentIndex(i);
+            }}
+            className={`h-2 w-2 rounded-full ${
+              i === currentIndex ? 'bg-green' : 'bg-gray-300'
+            }`}
+            aria-label={`Go to slide ${i + 1}`}
+          />
+        ))}
+      </div>
```

Comment:
we should be using a library for this which will handle responsive design, touch gestures, keyboard navigation, event callbacks, etc.